Amazon.IonObjectMapper.Test/NamingConventionTest.cs

```diff
@@ -34,8 +35,13 @@ public void CanConvertToSnakeCase()
         [TestMethod]
         public void CanUseTheNamingConventionOnAnObject()
         {
-            var stream = new IonSerializer(new IonSerializationOptions { NamingConvention = new SnakeCaseNamingConvention() }).Serialize(TestObjects.honda);
+            Dictionary<string, object> customContext = new Dictionary<string, object>() 
```

Comment:
If you use a new object, then you don't need to modify these tests.

Reply:
Fixed.